YoutubeScraper test fails

<!-- Before filing an issue, please search the existing issues (both open and closed) to see if your report might be duplicate -->
<!-- Please don't file an issue when you have a support question, send support questions to the mailinglist or ask them on Gitter (http://gitter.im/loklak/loklak) -->

- Issue type: Bug report

### Short description

Here is the log from build - 

```java
org.loklak.TestRunner > org.loklak.harvester.YoutubeScraperTest.parseFromInputStreamTest FAILED
    java.lang.AssertionError: 
    Expected: is "[\"Iggy Azalea - Team (Explicit) - YouTube\"]"
         but: was "[\"YouTube\"]"
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
        at org.junit.Assert.assertThat(Assert.java:956)
        at org.junit.Assert.assertThat(Assert.java:923)
        at org.loklak.harvester.YoutubeScraperTest.parseFromInputStreamTest(YoutubeScraperTest.java:39)
```

### Environment
<!-- Tell us about the environment -->
 - Operating system: Ubuntu 16.04LTS
 - Software version: 8d521a8461ef65a4b4ab5c8b90d5f5b43d0fbf23
 - Software source: Github repository

### Steps to reproduce
<!-- Tell us step-by-step how the issue can be triggered. Attach any screenshots if available -->
1. Get even with `loklak:development`.
2. `./gradlew build -i`
3. Look at the logs

### Expected behaviour
The test should pass.

### Actual behaviour
Test fails.

--- a/src/org/loklak/http/ClientConnection.java
+++ b/src/org/loklak/http/ClientConnection.java
@@ -78,14 +78,14 @@ import org.loklak.data.DAO;
  */
 public class ClientConnection {
 
-    public static String USER_AGENT = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.98 Safari/537.36";
+    public static String USER_AGENT = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.89 Safari/537.36";
 
     public  static final String CHARSET = "UTF-8";
     private static final byte LF = 10;
     private static final byte CR = 13;
     public static final byte[] CRLF = {CR, LF};
     private static final boolean debugLog = DAO.getConfig("flag.debug.redirect_unshortener", "false").equals("true");
-    
+
     private static RequestConfig defaultRequestConfig = RequestConfig.custom()
             .setSocketTimeout(60000)
             .setConnectTimeout(60000)
@@ -95,7 +95,7 @@ public class ClientConnection {
             .build();
 
     private final static CloseableHttpClient httpClient = getClosableHttpClient();
-    
+
     private int status;
     private BufferedInputStream inputStream;
     private CloseableHttpResponse httpResponse;
@@ -152,11 +152,11 @@ public class ClientConnection {
                 .setMaxConnTotal(500)
                 .build();
     }
-    
+
     public InputStream getInputStream() {
         return this.inputStream;
     }
-    
+
     /**
      * get a connection manager
      * @param trustAllCerts allow opportunistic encryption if needed
@@ -189,7 +189,7 @@ public class ClientConnection {
         cm.setDefaultMaxPerRoute(20);
         cm.setMaxPerRoute(new HttpRoute(new HttpHost("twitter.com", 80)), 50);
         cm.setMaxPerRoute(new HttpRoute(new HttpHost("twitter.com", 443)), 50);
-    	
+
         return cm;
     }
 
@@ -383,7 +383,7 @@ public class ClientConnection {
             this.inputStream = null;
         }
     }
-    
+
     public void finalize() {
         this.close();
     }
@@ -436,7 +436,7 @@ public class ClientConnection {
     public int getStatusCode() {
         return this.httpResponse.getStatusLine().getStatusCode();
     }
-    
+
     public static void main(String[] args) {
         // lets make a read-world test to find out which closing-methods cause less traffic
         String url[] = new String[]{

